Core/include/Acts/Vertexing/AdaptiveMultiVertexFinder.ipp

```diff
@@ -24,7 +24,8 @@ auto Acts::AdaptiveMultiVertexFinder<vfitter_t, sfinder_t>::find(
   // Seed tracks
   std::vector<const InputTrack_t*> seedTracks = allTracks;
 
-  FitterState_t fitterState(*m_cfg.bField, vertexingOptions.magFieldContext);
+  FitterState_t fitterState(*m_cfg.bField, vertexingOptions.magFieldContext,
+                            m_logger->cloneWithSuffix("State"));
```

Comment:
Why does the state need a logger? If you need this in state methods (which should be avoided anyway if possible I would say), you can pass in the tool logger as a const ref and use the log macros as usual.

Reply:
I will try to move the state methods to the main class - then I can use its logger too

Reply:
Moving the functions to the main class is also not pretty... I kept them in `state` for now, I think this should be dealt with in a separate PR

Thanks for the suggestions with the logger, I removed it from `state`